nrf70_bm_lib/docs/source/nrf70_bm_porting_guide.rst

```diff
@@ -89,6 +89,36 @@ For more information, visit the `nRF Connect SDK nrfxlib repository <https://git
 
    The git submodule pulls in the entire repo, but only `nrf_wifi` directory is required for the BM driver.
 
+TX power configuration
+**********************
+
+The maximum transmit power achieved on an nRF70 Series device-based product depends on the frequency band and operating channel.
+This varies across different Printed Circuit Board (PCB) designs.
+
+Multiple calibrations and checks are implemented to ensure consistency across channels and devices.
+However, these values depend on PCB design, which may result in Error Vector Magnitude (EVM) and spectral mask failures.
+To avoid this problem, you can specify the power ceiling at which the EVM and spectral mask are met for a given PCB design.
+
+The build-time parameters are made available to drivers through the `C` header file.
+The reference header files are located in the `nrf70_bm_lib/include` directory.
+
+The following files are used to configure the TX power:
+
+- `nrf70_tx_pwr_ceil_dk.h`: Contains the TX power configuration for the nRF7002 Development Kit (DK).
+
+- `nrf70_tx_pwr_ceil_ek.h`: Contains the TX power configuration for the nRF7002 Evaluation Kit (EK).
+
+For a new platform, you can create a new header file with the same structure as the reference files and replace the values with the appropriate values for the new platform.
+
+For more information on TX power calculation, refer to the `nRF70 TX power calculation <https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/drivers/wifi/nrf70_native.html#tx_power_calculation>`_.
```

Comment:
The problem here is that this link is in the latest NCS but some of the tx power overlay params etc pointed to in this link should be viewed for NCS 2.7.0 version. One reason is that tx power ceilings were x4 in 2.7.0 but x1 in 2.8.0/latest. We should somehow capture this subtlety

Reply:
Why - was it removed @krish2718 ?

Reply:
No, this section is brand new, will only be available in 2.9.0.